### PR TITLE
Feature request: markedDays props

### PR DESCRIPTION
**Pull request by bluenex**
Hi,

I'm looking for a React calendar library that is minimal and simple. This project looks very well suit to what I need. However, it lacks features I would like to use so here's my PR. I think it could be useful to have `markedDays` props so that we can mark days with events or something like that on the calendar. Here is an image of my implementation:

image

However, I think this will need some more tweaks:

- ~white mark color on today is looking better~
- ~how should we implement different mark colors on today and other days?~

plus:

- do you guys think it's good to add `onClick` prop? I'm thinking about clicking on the day and it returns the day being click, something like:

```js
onClick: day => doSomething(day)
```

This way when the user click we can hook the navigation function on it to the certain day.

Thank you for a very neat project again! 😄

### CODE REVIEW

**Comment by arncet** [comment]
Hi @bluenex, thank you for your feedback and your contribution.
`markedDays` is a good idea of feature.
I putted some comments on your PR.
Callback methods like `onClick` (or `onChangeMonth` ...) can be really helpful, i'll try to add those soon

File: src/components/Calendar.js [hunk 1]
```diff
@@ -101,7 +101,7 @@ class Calendar extends PureComponent {
         <div className="rlc-days">
           {getDays(firstDayToDisplay).map(day =>
             <div
-              className={`rlc-day ${this.getClassNames(day)}`}
+              className={`rlc-day ${this.getClassNames(day)}${markedDays && markedDays.map(x => getDateWithoutTime(x)).includes(day) ? ' marked-day' : ''}`}
```
Comment (arncet) [inline src/components/Calendar.js]: All classnames are generated by `getClassNames` method, so you can manage `marked-day` on this method

File: src/components/Calendar.js [hunk 2]
```diff
@@ -133,7 +134,8 @@ Calendar.propTypes = {
   displayTime: bool,
   dayLabels: arrayOf(string),
   monthLabels: arrayOf(string),
-  timezone: string
+  timezone: string,
+  markedDays: arrayOf(number)
```
Comment (arncet) [inline src/components/Calendar.js]: Don't forget to update `README.md`

Comment (arncet) [inline src/components/Calendar.js]: I think that `markedDays` can also be a function (`oneOfType([arrayOf(number), func])`), like `disableDates` for more flexibility.

**Comment by bluenex** [comment]
@arncet 

Hi,

Sorry for drag it for long, I almost forget about this PR. I've fixed all according to your suggestion. In the latest commit (7ee10f5), I added `onClickDate` prop which works fine to me but I'm not sure if it is ok for you. Could you please check it out?

Sorry again that this has been too long, I hope we can merge this soon!

**Comment by arncet** [comment]
Hi @bluenex, thank you for your feedback, your changes will soon be merged.

**Pull request merged**